.cbmc-batch/jobs/aws_cryptosdk_rsa_decrypt/aws_cryptosdk_rsa_decrypt_harness.c

```diff
@@ -21,31 +21,38 @@
 #define KEY_LEN 256
 
 void aws_cryptosdk_rsa_decrypt_harness() {
+    /* Nondet Inputs */
     struct aws_byte_buf plain;
     struct aws_allocator *alloc = can_fail_allocator();
     struct aws_byte_cursor cipher;
     struct aws_string *key = ensure_string_is_allocated_bounded_length(KEY_LEN);
     enum aws_cryptosdk_rsa_padding_mode rsa_padding_mode;
 
+    /* Assumptions */
+    __CPROVER_assume(key != NULL);
```

Comment:
Why can't we assume here `aws_string_is_valid()`?

Reply:
Updated and also using the new c-common functions.